doc/source/conf.py

```diff
@@ -150,7 +150,9 @@ def _stop_fluent_container(gallery_conf, fname):
     # path where to save gallery generated examples
     "gallery_dirs": ["examples"],
     # Pattern to search for example files
-    "filename_pattern": r"\.py",
+    "filename_pattern": r"exhaust_system_settings_api\.py|external_compressible_flow\.py|mixing_elbow_settings_api\.py|modeling_cavitation\.py",
+    # Disabled example scripts
+    "ignore_pattern": r"ahmed_body_workflow\.py|brake\.py|DOE_ML\.py|radiation_headlamp\.py",
```

Comment:
@mkundu1 

We have specified both patterns therefore it is building the existing 4 examples only. 

![image](https://github.com/user-attachments/assets/cf6186b4-7d4a-44c0-b20b-9b15949279b3)